### PR TITLE
nix3-run: add more examples

### LINKED ISSUE #10619: page request: nix3 run
### Command description

Today the page of this command only contains two examples when this command is widely used by the Nix community and has a lot of more uses 

### Command details

_No response_

### OS

- [ ] Android
- [X] Linux
- [X] macOS
- [ ] sunOS
- [X] Windows

### PR DESCRIPTION
**Pull request by luisnquin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

resolves #10619

### CODE REVIEW

File: pages/common/nix3-run.md [hunk 1]
```diff
@@ -1,13 +1,33 @@
 # nix run
 
-> Run an application from a Nix flake.
+> Run an application from a **Nix flake**.
```
Comment (kbdharun) [inline pages/common/nix3-run.md:3]:
```suggestion
> Run an application from a Nix flake.
```
Our clients don't support rendering Bold or Italic.

image

File: pages/common/nix3-run.md [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+- Run providing arguments to the program:
 
-`nix run`
+`nix run nixpkgs#vim -- path/to/file`
+
+- Run from a remote repository:
+
+`nix run {{remote_name}}:{{owner}}/{{repo}}`
+
+- Run from a remote repository using an specific tag, revision or branch:
```
Comment (kbdharun) [inline pages/common/nix3-run.md:23]:
```suggestion
- Run from a remote repository using a specific tag, revision or branch:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by luisnquin** [comment]
Done @kbdharun

**Review by kbdharun — APPROVED** [review]

**Review by imlucyd — APPROVED** [review]

File: pages/common/nix3-run.md [hunk 3]
```diff
@@ -4,10 +4,30 @@
...
+- Run the default application in the flake in the current directory:
+
+`nix run`
+
 - Run a command whose name matches the package name from nixpkgs (if you want a different command from that package, see `tldr nix3 shell`):
 
 `nix run nixpkgs#{{pkg}}`
 
-- Run the default application in the flake in the current directory:
+- Run providing arguments to the program:
```
Comment (acuteenvy) [inline pages/common/nix3-run.md:15]:
```suggestion
- Run a command with provided arguments:
```

File: pages/common/nix3-run.md [hunk 4]
```diff
@@ -4,10 +4,30 @@
...
 
-`nix run`
+`nix run nixpkgs#vim -- path/to/file`
```
Comment (acuteenvy) [inline pages/common/nix3-run.md:17]:
```suggestion
`nix run nixpkgs#{{vim}} -- {{path/to/file}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**